README.md

```diff
@@ -255,6 +255,32 @@ charts:
   - ...
 ~~~
 
+#### Amazon Elastic Container Registry (ECR)
+Amazon Elastic Container Registry (ECR) is an OCI registry, but it has two peculiarities that should be taken into account when interacting with charts-syncer.
+
+The first peculiarity relates to authentication, usually you would have an IAM user with access_key_id and secret_access_key credentials. These **are not** the credentials that should be entered in the config file.
+To obtain the proper credentials, you need to obtain a temporary password to operate the ECR registry. By using the following command, you can get the password:
+
+```
+$ aws ecr get-login-password --region REGION
+```
+
+In the previous command remember to update the REGION placeholder by a proper value.
+The username will always be `AWS`.
+
+The second peculiarity only affects when using ECR as a target registry since you need to create the repositories in advance.
+If the repositories don't exist when executing charts-syncer you will get an error like the following:
+
+```
+failed to do request: Post "https://AWS_ACCOUNT.dkr.ecr.AWS_REGION.amazonaws.com/v2/charts-syncer-test/charts/common/blobs/uploads/": EOF
+failed to do request: Post "https://AWS_ACCOUNT.dkr.ecr.AWS_REGION.amazonaws.com/v2/charts-syncer-test/charts/apache/blobs/uploads/": EOF
+failed to do request: Post "https://AWS_ACCOUNT.dkr.ecr.AWS_REGION.amazonaws.com/v2/charts-syncer-test/charts/mariadb/blobs/uploads/": EOF
+failed to do request: Post "https://AWS_ACCOUNT.dkr.ecr.AWS_REGION.amazonaws.com/v2/charts-syncer-test/charts/mysql/blobs/uploads/": EOF
+failed to do request: Post "https://AWS_ACCOUNT.dkr.ecr.AWS_REGION.amazonaws.com/v2/charts-syncer-test/charts/redis/blobs/uploads/": EOF
```

Comment:
is this AWS or our tool? If the latter, we should probably try to capture the real error and show it.

Reply:
This is our tool. By capturing you mean to show a nice error message? If so, I would avoid doing it since this is specific for ECR (which is not an input type since we use OCI). We would need to parse the registry URL to see if it looks like an ECR URL just to guess if it is.